tests/integration/telemetry/stackdriver/vm/vm_test.go

```diff
@@ -147,13 +150,48 @@ spec:
 					}
 				}
 
+				// Verify edges
+				edges, err := sdInst.ListTrafficAssertions()
+				if err != nil {
+					return fmt.Errorf("failed to get traffic assertions: %v", err)
+				}
+
+				expectedEdge, err := expectedTrafficAssertion()
+				if err != nil {
+					return fmt.Errorf("failed to get wanted traffic assertion: %v", err)
+				}
+
+				foundEdge := false
+				for _, ta := range edges {
+					srcUID := ta.Source.Uid
+					dstUID := ta.Destination.Uid
+
+					ta.Source.Location = ""
+					ta.Source.ClusterName = ""
+					ta.Source.Uid = ""
+					ta.Destination.Uid = ""
+
+					if diff, equal := messagediff.PrettyDiff(ta, expectedEdge); !equal {
+						fmt.Printf("Traffic Assertion Diff: %v", diff)
```

Comment:
Use istio log pkg instead of fmt.Printf?

Reply:
Done.